pkgs/by-name/ti/tinyxxd/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tinyxxd";
+  version = "1.3.3";
+  src = fetchFromGitHub {
+    repo = "tinyxxd";
+    owner = "xyproto";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-mup4xHt5Prpcwz2QU0qz+9/KYOE0HNrLx9b7z5o47ag=";
+  };
+
+  outputs = [
+    "out"
+    "xxd"
+  ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+
+  postInstall = ''
+    mkdir -p $xxd/bin
+    ln -s $out/bin/tinyxxd $xxd/bin/xxd
+  '';
+
+  meta = {
+    homepage = "https://github.com/xyproto/tinyxxd";
+    description = "Drop-in replacement and standalone version of the hex dump utility that comes with ViM";
+    license = lib.licenses.gpl2;
```

Comment:
This needs to be `lib.licenses.gpl2Only` or `lib.licenses.gpl2Plus`. The Arch package implies the latter, but the repository seems to mostly imply the former. It’s also lacking a credit for the original author Juergen Weigert. I will raise these concerns upstream and we should resolve them before merging this.

Reply:
Added credits.

The dual license from ViM is not 100% clear if GPL-2.0 is `GPL-2.0-only` or `GPL-2.0-or-later`.